From 27c2de7efd350933dc4f24645fdc76899b18400f Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Fri, 9 Jun 2023 21:03:07 -0400
Subject: [PATCH 1910/1936] Remove --no-as-needed flag

This flag broke the build on Clang and is a workaround for a no longer
relevant version of gcc.
---
 src/Tests/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Tests/CMakeLists.txt b/src/Tests/CMakeLists.txt
index 3204a565..f221ddc3 100644
--- a/src/Tests/CMakeLists.txt
+++ b/src/Tests/CMakeLists.txt
@@ -16,7 +16,7 @@ add_definitions(-DSOURCE_DIR="${CMAKE_CURRENT_SOURCE_DIR}")
 #Extra libraries added to make test and full compilation use the same library
 #links for quirky compilers
 set(test_lib zynaddsubfx_core ${GUI_LIBRARIES} ${ZLIB_LIBRARY} ${FFTW3F_LIBRARIES}
-    ${MXML_LIBRARIES} pthread "-Wl,--no-as-needed -lpthread")
+    ${MXML_LIBRARIES} pthread )
 
 message(STATUS "Linking tests with: ${test_lib}")
 
-- 
2.47.0

